thirdparty/build-thirdparty.sh

```diff
@@ -1238,6 +1238,20 @@ build_clucene() {
     fi
 }
 
+# avx2ki
+build_avx2ki() {
+    check_if_source_exist $AVX2KI_SOURCE
+    cd $TP_SOURCE_DIR/$AVX2KI_SOURCE
+    rpm2cpio boostkit-ksl-2.0.0-1.aarch64.rpm | cpio -id
```

Comment:
not sure if rpm2cpio is available on ubuntu platform, may need to install the toolkit in dev-env for ubuntu platform.

Reply:
How this should be done?

Reply:
need to make changes in dev-env.Dockerfile, make sure rpm2cpio deb package installed in the environment.